### PR TITLE
simple animation and re-styling for all buttons

### PR DESCRIPTION
**Pull request by 0xmafan**
Hey Chris, I'm a newbie coder learning from Scrimba, just like you. And this pull request is not an important thing, I just add a little bit of styling and animation to buttons in your beautiful code, just to exercise CSS and git a little more. I appreciate it if you accept this :)
Happy coding, cheers

### CODE REVIEW

**Comment by devtopher** [comment]
Moharram,

I'm honored that my little chrome extension is your first pull request! I will check it out and let you know! Thanks so much!

File: index.css [hunk 1]
```diff
@@ -85,14 +101,15 @@ a {
 }
 
 .delbtn {
-    color: red;
-    background-color:white;
-    float:right;
```
Comment (devtopher) [inline index.css:90]: Without float:right all the delete buttons line up right next to the billNames. I think I prefer the smaller delete button and having it on the right side.

File: index.css [hunk 2]
```diff
@@ -30,16 +30,34 @@ h1{
...
-    background:#3d9cb1;
-    color: #0b2b44;
+button:hover,
+button:focus {
+    color: #FFF;
+    outline: 0;
+}
+
+.primary {
+    transition: box-shadow 300ms ease-in-out, color 300ms ease-in-out;
```
Comment (devtopher) [inline index.css]: What do you think about speeding up the transition on all of the button effects. I don't mind the mouseover effect, it just seems slow to me.

File: index.css [hunk 3]
```diff
@@ -30,16 +30,34 @@ h1{
     font-weight: bold;
 }
 button {
-    background: #3d9cb1;
```
Comment (devtopher) [inline index.css:33]: I think I prefer the darker background for the 3 primary buttons. Do you want to see if you can find a hover state that works well with keeping the darker background on the initial state?

**Review by devtopher — CHANGES_REQUESTED** [review]
Thanks so much for working on this. I made a few notes about things that might be good to change. If you have any thoughts or disagreements let me know.

File: index.css (hunk 1 above)
Comment (0xmafan) [reply index.css:90]: Okay, honestly I'm not so sure what should I do now.
I just changed this one and pushed it again.
I'm going for the other 2 later.

File: index.css (hunk 2 above)
Comment (0xmafan) [reply index.css]: I speed up the buttons too. I think 100ms is fast enough, anything less than that doesn't feel like animation. Unless you don't want animation at all :))

File: index.css (hunk 3 above)
Comment (0xmafan) [reply index.css:33]: Actually, I don't understand this well. Do you mean you don't want those 3 buttons to be transparent?
I really have no idea how to choose colors for the initial state and the hover state. If you suggest the colors I will fix it. 

*By the way, in my opinion, when all the buttons are transparent it has a nice and unified look. But it's your code, so you decide :)

Comment (devtopher) [reply index.css:33]: I meant basically make the :hover the opposite of what it is now. So the button is solid to start and transparent on hover. Not sure if that will look good but I feel the transparent is a little hard to read

Comment (0xmafan) [reply index.css:33]: Ok, now I understand. Sorry, English is my 3rd language and I'm not very good at it.
Anyway, I fixed it. See if it's what you want?

**Comment by devtopher** [comment]
I like all the changes. Thanks for the help with styling the buttons and adding animations. I appreciate you taking part in the code for this billtracker!

**Comment by 0xmafan** [comment]
Thanks, It's a great experience.

**Pull request merged**